src/relentless/simulate/dilute.py

```diff
@@ -171,43 +171,54 @@ def process(self, sim, sim_op):
             ens.rdf[pair] = ensemble.RDF(
                 sim.potentials.pair.linear_space, numpy.exp(-u / kT)
             )
-
         # compute pressure
-        ens.P = 0.0
-        for a in sim.types:
-            rho_a = ens.N[a] / ens.V.extent
-            ens.P += kT * rho_a
-            for b in sim.types:
-                rho_b = ens.N[b] / ens.V.extent
-                r = sim.potentials.pair.linear_space
-                u = sim.potentials.pair.energy((a, b))
-                f = sim.potentials.pair.force((a, b))
-                gr = ens.rdf[a, b].table[:, 1]
-
-                # only count continuous range of finite values
-                flags = numpy.isinf(u)
-                first_finite = 0
-                while flags[first_finite] and first_finite < len(r):
-                    first_finite += 1
-                r = r[first_finite:]
-                u = u[first_finite:]
-                f = f[first_finite:]
-                gr = gr[first_finite:]
-
-                if sim.dimension == 3:
-                    geo_prefactor = 4 * numpy.pi * r**2
-                elif sim.dimension == 2:
-                    geo_prefactor = 2 * numpy.pi * r
-                else:
-                    raise ValueError(
-                        "Geometric integration factor unknown for extent type"
-                    )
-                y = (geo_prefactor / 6.0) * rho_a * rho_b * f * gr * r
-                ens.P += numpy.trapz(y, x=r)
-
-        sim[self]["ensemble"] = ens
-        sim[self]["num_thermo_samples"] = None
-        sim[self]["num_rdf_samples"] = None
+        B = 0.0
+        N = sum(ens.N[i] for i in sim.types)
```

Comment:
I like to put summation variables right before the loop with separation, but that's just a matter of style preference
```suggestion
        N = sum(ens.N[i] for i in sim.types)

        B = 0.0
```